### DISCUSSION TITLE
Antenna option documentation?

### DISCUSSION THREAD

**Discussion by zerog2k** [gdyuldin/x6100_gui#58] [Q&A] [answered]
What is the "Antenna" value? 
Are these like memory banks for recalling per-frequency tuner settings for different antenna profiles?

**Comment by gdyuldin** [comment, marked as answer]
Looks like Oleg's (R1CBU) idea was to save/load ATU values for each antenna you use with the device. 
https://github.com/gdyuldin/x6100_gui/blob/main/src/params/params.c#L746
